File: Day236_2025/Q236_GFG.java
import java.util.Arrays;

class Q236_GFG {
    public int minDaysBloom(int[] arr, int k, int m) {
        Arrays.sort(arr);
        if (arr.length < m * k) return -1;
        return arr[m * k - 1];
    }
}

File: Day236_2025/Q236_GFG.cpp
class Solution {
  public:
    int minDaysBloom(vector<int>& arr, int k, int m) {
        // Q. Minimum days to make M bouquets
        // Code here
        sort(arr.begin(), arr.end());
        if ( arr.size() <  m * k) return -1;
        return arr[m * k-1];
    }
};